doc/releases/quincy.rst

```diff
@@ -0,0 +1,3614 @@
+  For more detail, see:
+
+  https://docs.ceph.com/en/latest/rados/operations/placement-groups/
+
+* MGR: The pg_autoscaler can now be turned `on` and `off` globally
+  with the `noautoscale` flag. By default this flag is unset and
+  the default pg_autoscale mode remains the same.
+  For more detail, see:
+
+  https://docs.ceph.com/en/latest/rados/operations/placement-groups/
+
+* MON/MGR: Pools can now be created with the `--bulk` flag. Any pools created 
+  with `bulk` will use a profile of the `pg_autoscaler` that provides more 
+  performance from the start. However, pools that were created without the 
+  `--bulk` flag use the old behavior by default. For more detail, see:
+
+  https://docs.ceph.com/en/latest/rados/operations/placement-groups/
+
+
+RBD block storage
+~~~~~~~~~~~~~~~~~
+  
+## TODO
+
+RGW object storage
+~~~~~~~~~~~~~~~~~~
+
+* RGW now supports rate limiting by user and/or by bucket. With this
+  feature it is possible to limit user and/or bucket, the total operations
+  and/or bytes per minute can be delivered. This feature allows the
+  admin to limit only READ operations and/or WRITE operations. The
+  rate-limiting configuration could be applied on all users and all buckets
+  by using global configuration.
+
+* `radosgw-admin realm delete` has been renamed to `radosgw-admin realm
+  rm`. This is consistent with the help message.
+
+* S3 bucket notification events now contain an `eTag` key instead of
+  `etag`, and eventName values no longer carry the `s3:` prefix, fixing
+  deviations from the message format that is observed on AWS.
+
+* It is possible to specify ssl options and ciphers for beast frontend
+  now. The default ssl options setting is
+  "no_sslv2:no_sslv3:no_tlsv1:no_tlsv1_1". If you want to return to the old
+  behavior, add 'ssl_options=' (empty) to the ``rgw frontends`` configuration.
+
+* The behavior for Multipart Upload was modified so that only 
+  CompleteMultipartUpload notification is sent at the end of the multipart 
+  upload. The POST notification at the beginning of the upload and the PUT 
+  notifications that were sent on each part are no longer sent.
+
+
+CephFS distributed file system
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* fs: A file system can be created with a specific ID ("fscid"). This is
+  useful in certain recovery scenarios (for example, when a monitor
+  database has been lost and rebuilt, and the restored file system is
+  expected to have the same ID as before).
+
+* fs: A file system can be renamed using the `fs rename` command. Any cephx
+  credentials authorized for the old file system name will need to be
+  reauthorized to the new file system name. Since the operations of the clients
+  using these re-authorized IDs may be disrupted, this command requires the
+  "--yes-i-really-mean-it" flag. Also, mirroring is expected to be disabled
+  on the file system.
+
+* MDS upgrades no longer require all standby MDS daemons to be stoped before
```

Comment:
@vshankar does this apply to pacific -> quincy, or just to future upgrades post-quincy?

Reply:
This is available from pacific v16.2.6::

```
commit f2a16a7345125d7732a0128c7df71ec10e78f28a
Author: Patrick Donnelly <pdonnell@redhat.com>
Date:   Tue Sep 14 11:12:10 2021 -0400

    doc: note simpler MDS upgrade procedure for v16.2.6
    
    Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

diff --git a/doc/releases/pacific.rst b/doc/releases/pacific.rst
index 4f07f8c5ea6..c53cfea2073 100644
--- a/doc/releases/pacific.rst
+++ b/doc/releases/pacific.rst
@@ -20,6 +20,13 @@ Notable Changes
   For more details, see:
   https://docs.ceph.com/en/latest/rados/operations/placement-groups/
 
+* CephFS: the upgrade procedure for CephFS is now simpler. It is no longer
+  necessary to stop all MDS before upgrading the sole active MDS. After
+  disabling standby-replay, reducing max_mds to 1, and waiting for the file
+  systems to become stable (each fs with 1 active and 0 stopping daemons), a
+  rolling upgrade of all MDS daemons can be performed.
```

So, it does apply for pacific -> quincy upgrades.